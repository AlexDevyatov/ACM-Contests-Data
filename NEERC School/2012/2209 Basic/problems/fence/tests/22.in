11
-49 -5
-42 -42
-39 -52
14 -54
44 -55
55 -16
55 45
9 55
-8 55
-30 54
-47 49
-27 20
