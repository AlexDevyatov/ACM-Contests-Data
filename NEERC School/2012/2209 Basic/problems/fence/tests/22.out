12
-49 -5
-42 -42
-27 20
-39 -52
14 -54
44 -55
55 -16
55 45
9 55
-8 55
-30 54
-47 49
